test/image-blueprints/group2/rhel92-source-fake-next-next-minor.toml

```diff
@@ -0,0 +1,39 @@
+name = "rhel-9.2-microshift-source-fake-next-next-minor"
```

Comment:
This name is a bit subtle when scanning. Can we use YPLUS2 or some other more distinct thing other than "next next"?

Reply:
Yes, thanks for a suggestion, I had no ideas at the moment of writing :stuck_out_tongue_winking_eye: